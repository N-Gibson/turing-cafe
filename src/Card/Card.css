.card {
  background-color: #fff;
  border: 4px solid #008080;
  border-radius: 20px;
  height: 250px;
  width: 200px;
  margin: 10px;
}