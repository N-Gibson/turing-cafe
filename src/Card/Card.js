import React from 'react';
import './Card.css';

const Card = (reservation) => {
  const { name, date, time, id, number } = reservation.reservation;

  return (
    <div className='card'>
      <h2>{name}</h2>
      <p>{date}</p>
      <p>{time} pm</p>
      <p>Number of guests: {number}</p>
      <button>Cancel</button>
    </div>
  )
}

export default Card;