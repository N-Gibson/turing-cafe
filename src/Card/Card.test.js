import React from 'react';
import { shallow } from 'enzyme';
import Card from './Card';

describe('Form', () => {
  let wrapper, res;
  beforeEach(() => {
    reservation = {
      reservation: {
        date: "12/29",
        id: "1",
        name: "Christie",
        number: "12",
        time: "7:00",
      }
    }
    wrapper = shallow(<Card reservation={res}/>)
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})