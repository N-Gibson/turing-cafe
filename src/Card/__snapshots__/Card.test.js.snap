// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should match snapshot 1`] = `
<div
  className="card"
>
  <h2 />
  <p />
  <p>
     pm
  </p>
  <p>
    Number of guests: 
  </p>
  <button
    onClick={[Function]}
  >
    Cancel
  </button>
</div>
`;
